app/controllers/home_controller.rb

```diff
@@ -47,6 +47,17 @@ def dashboard
     end
   end
 
+  def dashboard_v2
+    # The new dashboard displays the blog and topics list
+    if current_user
+      @blog = Tag.find_nodes_by_type('blog', 'note', 1).first
```

Comment:
Here we could add .limit(1) to optimize the query a bit.
```suggestion
      @blog = Tag.find_nodes_by_type('blog', 'note', 1).limit(1).first
```

Reply:
Alright 👌🏾

Reply:
Done!